src/main/scripts/org/reaktivity/specification/nukleus/tcp/streams/rfc793/server.sent.reset.then.end/client.rpt

```diff
@@ -0,0 +1,29 @@
+#
```

Comment:
For consistency with the other scripts please rename the folded to "server.sent.reset.and.end" (replacing the word then also has the advantage of no longer implying that ordering is significant for reset and end)

Reply:
Done.